tensorzero_internal/src/clickhouse.rs

```diff
@@ -34,32 +36,36 @@ impl ClickHouseConnectionInfo {
     /// However, for tests that directly test ClickHouse behavior, you can directly create the struct.
     pub async fn new(database_url: &str) -> Result<Self, Error> {
         // Add a query string for the database using the URL crate
-        let mut database_url = Url::parse(database_url).map_err(|e| {
+        let parsed_url = Url::parse(database_url).map_err(|e| {
             Error::new(ErrorDetails::Config {
                 message: format!("Invalid ClickHouse database URL: {}", e),
```

Comment:
This isn't an issue with your PR, but we shouldn't be printing this. Let's replace every mention of `Invalid ClickHouse database URL: {}` with `Invalid ClickHouse database URL.` (there are multiple mentions)